### ISSUE TITLE
icon on notification not displayed

### ISSUE THREAD

**Issue by Storik4pro** [ysfchn/toasted#3]
I register the application through the example function and do not see the icon.
```
def run_example():
    example_register_custom_app_id("Lines.Storik4.goodbyedpiUI", "GoodbyeDPI UI", 'file:///C:/Users\serst\Pictures/asdasd.png')
    asyncio.run(show_battery_example("Lines.Storik4.goodbyedpiUI"))
```
And app is registering
![image](https://github.com/user-attachments/assets/90d5271f-2244-4393-8599-df6c4a53f6d1)
But, i'm dont seen icon on nonflication
![image](https://github.com/user-attachments/assets/c3c864b6-cfc3-4b6e-8828-d3da46b72feb)
What am I doing wrong?

**Comment by ysfchn** [comment]
Since you just want to use a local image on your computer as an app icon, you can just register the app ID with `Toast.register_app_id` normally. The snippet given in the example script is just a shortcut method to use a built-in Windows icon as an app icon. Otherwise, `Toast.register_app_id` is the recommended way to register an app ID.

To register an app ID:

```
Toast.register_app_id("Lines.Storik4.goodbyedpiUI", "GoodbyeDPI UI", icon_uri = "C:/Users/serst/Pictures/asdasd.png")
```

Note that `register_app_id` method doesn't require the URI scheme (so no `file:///` must be used for this one), just the path of the image file where it is located on the computer.

Let me know if you need further help.

**Comment by Storik4pro** [comment]
I changed the way I register the app, but the icon still doesn't appear in the notification. 
```
def register_app():
    Toast.unregister_app_id('Lines.Storik4.goodbyedpiUI')
    Toast.register_app_id("Lines.Storik4.goodbyedpiUI", "GoodbyeDPI UI", icon_uri = "E:\ByeDPI\data\icon.png")
    example_print_app_ids()
```
I also tried an existing control panel link as a link to the icon
```
def register_app():
    Toast.unregister_app_id('Lines.Storik4.goodbyedpiUI')
    Toast.register_app_id("Lines.Storik4.goodbyedpiUI", "GoodbyeDPI UI", icon_uri = "%SystemRoot%\ImmersiveControlPanel\images\logo.png")
    example_print_app_ids()
```
But this had no effect. 
I also tried converting python script to exe, but it didn't help

**Comment by Storik4pro** [comment]
I was able to fix the problem. In "app_id" you should specify the id without "."
Examples:
`asyncio.run(show_message("Lines.Storik4.goodbyedpiUI", title, message))`

![image](https://github.com/user-attachments/assets/67b1da43-f437-4da9-ba6d-5bc2900e6ba0)

`asyncio.run(show_message("GoodbyeDPI_app", title, message))`

![image](https://github.com/user-attachments/assets/0905998f-92be-47de-905d-73c9177b3d64)

In this case, both keys exist in the registry with the same values

![image](https://github.com/user-attachments/assets/9872f908-b8c8-4010-b2ce-70b80e5d0cd2)
![image](https://github.com/user-attachments/assets/00ca3c30-f3ab-4127-a8f9-565e65329b59)

**Comment by ysfchn** [comment]
App IDs are allowed to contain dots (.) so I don't think that was the problem. I guess it was caused by some sort of Windows caching (maybe a restart can resolve it?) or something else, I'm glad that you are able to resolve it, I will look further into the issue just in case.

**Comment by Storik4pro** [comment]
Yes, rebooting solves the problem. I didn't think about it at all. Thank you! 
But, there is also an unpleasant problem that I encountered on another PC during testing. This is an error when writing the application to the registry

![image](https://github.com/user-attachments/assets/d349adac-0c3b-4aa1-a966-9bbd7be59ca3)

`WinError 1021 "Cannot create a stable subkey under a volatile parent key."` can occur if there is already an application on the system that registers the `AppUserModelId` as temporary. In my case it was Windows Explorer

![image](https://github.com/user-attachments/assets/0ec86c40-a12d-4455-a32b-e0a4454cc0d3)

I tried to manually create a partition but received the error `"The partition cannot be created. There was an error writing to the registry"`

![image](https://github.com/user-attachments/assets/b6e06c0b-310c-4431-aa17-52b8c66f53a6)

The problem was solved by completely manually deleting the `AppUserModelId` parameter and creating it again as a static parameter.

I don't know if this problem can be solved automatically, but I thought it was worth writing about it here
